src/labels/CustomLabels.labels

```diff
@@ -6636,10 +6636,10 @@ Selecting Don't Auto Schedule Default NPSP Jobs tells NPSP not to check for or r
         <protected>false</protected>
         <shortDescription>stgHelpAdvancedMapping4</shortDescription>
         <value>
-            We couldn't convert some of your existing Help Text mappings because they were invalid. Read more about &lt;a href=&quot;https://powerofus.force.com/&quot; target=&quot;_blank&quot;&gt;common Help Text mapping issues&lt;/a&gt;. &lt;b&gt;Please take a screenshot of this page or copy the information in the table below to track updates you need to make to these mappings.&lt;/b&gt;
+            **This message will no longer be displayed if you leave this page!** We couldn't convert some of your existing Help Text mappings because they are invalid. Read more about &lt;a href=&quot;https://powerofus.force.com/&quot; target=&quot;_blank&quot;&gt;common Help Text mapping issues&lt;/a&gt;. &lt;b&gt;Please take a screenshot of this page or copy the information in the table below to track updates you need to make to these mappings.&lt;/b&gt;
```

Comment:
```suggestion
            **This message will no longer be displayed if you leave this page!** We couldn't convert some of your existing Help Text mappings because they are invalid. Read more about &lt;a href=&quot;https://powerofus.force.com/NPSP_Help_Text_Mapping&quot; target=&quot;_blank&quot;&gt;common Help Text mapping issues&lt;/a&gt;. &lt;b&gt;Please take a screenshot of this page or copy the information in the table below to track updates you need to make to these mappings.&lt;/b&gt;
```